specification/domainservices/resource-manager/Microsoft.AAD/stable/2021-05-01/domainservices.json

```diff
@@ -0,0 +1,1113 @@
+          "x-ms-enum": {
+            "name": "SyncOnPremPasswords",
+            "modelAsString": true
+          }
+        },
+        "kerberosRc4Encryption": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "A flag to determine whether or not KerberosRc4Encryption is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "KerberosRc4Encryption",
+            "modelAsString": true
+          }
+        },
+        "kerberosArmoring": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Disabled",
+          "description": "A flag to determine whether or not KerberosArmoring is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "KerberosArmoring",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "ConfigDiagnostics": {
+      "description": "Configuration Diagnostics",
+      "properties": {
+        "lastExecuted": {
+          "type": "string",
+          "format": "date-time-rfc1123",
+          "description": "Last domain configuration diagnostics DateTime"
+        },
+        "validatorResults": {
+          "type": "array",
+          "description": "List of Configuration Diagnostics validator results.",
+          "items": {
+            "$ref": "#/definitions/ConfigDiagnosticsValidatorResult"
+          }
+        }
+      }
+    },
+    "ConfigDiagnosticsValidatorResult": {
+      "description": "Config Diagnostics validator result data",
+      "properties": {
+        "validatorId": {
+          "type": "string",
+          "description": "Validator identifier"
+        },
+        "replicaSetSubnetDisplayName": {
+          "type": "string",
+          "description": "Replica set location and subnet name"
+        },
+        "status": {
+          "type": "string",
+          "enum": [
+            "None",
+            "Running",
+            "OK",
+            "Failure",
+            "Warning",
+            "Skipped"
+          ],
+          "default": "None",
+          "description": "Status for individual validator after running diagnostics.",
+          "x-ms-enum": {
+            "name": "status",
+            "modelAsString": true
+          }
+        },
+        "issues": {
+          "type": "array",
+          "description": "List of resource config validation issues.",
+          "items": {
+            "$ref": "#/definitions/ConfigDiagnosticsValidatorResultIssue"
+          }
+        }
+      }
+    },
+    "ConfigDiagnosticsValidatorResultIssue": {
+      "description": "Specific issue for a particular config diagnostics validator",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Validation issue identifier."
```

Comment:
>Validation issue identifier [](http://example.com/codeflow?start=26&length=27)

Suggest describing the format (e.g. guid, name, Url) so customer can find it in the docs. Not required for ARM signoff, just a suggestion.